pages/task/read/views/pil.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { Fragment } from 'react';
+import moment from 'moment';
+import { Snippet } from '@asl/components';
+import { dateFormat, procedureDefinitions } from '../../../../constants';
+import { connect } from 'react-redux';
+
+const Pil = ({ profile }) => {
+  const pil = profile.pil;
+  return (
+    <Fragment>
+      <h2><Snippet>pil.training.title</Snippet></h2>
+      { profile.certificates.length && profile.certificates.map((certificate, index) => (
+        <div className="certificate" key={index}>
+          <Fragment>
+            <h3><Snippet>pil.training.certificate.details</Snippet></h3>
+            <Snippet>pil.training.certificate.number</Snippet><span>:</span> {certificate.certificateNumber}<br/>
+            <Snippet>pil.training.certificate.awarded</Snippet><span>:</span> {moment(certificate.passDate, 'YYYY-MM-DD').format(dateFormat.short)}<br/>
+            <Snippet>pil.training.certificate.expiry</Snippet><span>:</span> {moment(certificate.passDate, 'YYYY-MM-DD').add(5, 'years').format(dateFormat.short)}<br/>
+            <Snippet>pil.training.certificate.body</Snippet><span>:</span> {certificate.accreditingBody}<br/>
```

Comment:
Why are we hard-coding a date format, _and_ using a constant?

Reply:
also, we have the lib `date-fns` for use in templates - moment is mahoosive so we removed it from anything that gets bundled